src/CMakeLists.txt

```diff
@@ -5,25 +5,27 @@ set(MAIN_SRC_FILES CovidSim.cpp BinIO.cpp Rand.cpp Error.cpp Dist.cpp Kernels.cp
 set(MAIN_HDR_FILES CovidSim.h BinIO.h Rand.h Constants.h Country.h Error.h Dist.h Kernels.h Bitmap.h Model.h Param.h SetupModel.h ModelMacros.h InfStat.h CalcInfSusc.h Sweep.h Update.h MicroCellPosition.hpp Direction.hpp InverseCdf.h)
 source_group(covidsim\\main FILES ${MAIN_SRC_FILES} ${MAIN_HDR_FILES})
 
+add_subdirectory(Geometry)
```

Comment:
Can we put all `add_subdirectory()` calls together as far as possible.  I don't think the ordering here is important.

Reply:
Moved back into its original position as the geometry binary isn't added to the exe until further down the file.